Dockerfile

```diff
@@ -1,66 +1,91 @@
-# Build Stage: using Go 1.24.1 image
+## Minimal runtime Dockerfile (microdnf-only, no torch, wrapper in site-packages)
+# Build stage
 FROM quay.io/projectquay/golang:1.25 AS builder
 ARG TARGETOS
 ARG TARGETARCH
 
-# Install build tools
-# The builder is based on UBI8, so we need epel-release-8.
 RUN dnf install -y 'https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm' && \
-    dnf install -y gcc-c++ libstdc++ libstdc++-devel clang zeromq-devel pkgconfig && \
+    dnf install -y gcc-c++ libstdc++ libstdc++-devel clang zeromq-devel pkgconfig python3.12-devel python3.12-pip git && \
```

Comment:
Q: not familiar with python packages. Is `-devel` needed or just a runtime package? I guess the `-devel` is for writing extensions in other languages?